charm/README.md

```diff
@@ -0,0 +1,51 @@
+# The Charm for the charmhub.io website
+
+This charm was created using the [PaaS App Charmer](https://juju.is/docs/sdk/paas-charm)
+
+## Local development
+
+To work on this charm locally, you first need to set up an environment, follow [this section](https://juju.is/docs/sdk/write-your-first-kubernetes-charm-for-a-flask-app#heading--set-things-up) of the tutorial.
+
+Then, you can run the following command to pack and upload the rock:
+
+```bash
+rockcraft pack
+rockcraft.skopeo --insecure-policy copy --dest-tls-verify=false oci-archive:charmhub-io*.rock docker://localhost:32000/charmhub-io:1
```

Comment:
for some engineers this will be the first time they interact with rocks - can you add a little explanation of these commands below (or links to the appropriate docs) thanks!